.meteor/packages

```diff
@@ -190,3 +190,4 @@ assistify:defaults
 chatpal:search
 rocketchat:version-check
 meteorhacks:aggregate
+deepl:translation
```

Comment:
the first part of the package should denote the author. We can make it `assistify:deepl-translation`

Reply:
Changed the author info to Assistify